ansible/roles/netbox/handlers/main.yml

```diff
@@ -17,3 +17,10 @@
     state: 'restarted'
     enabled: True
   when: netbox__app_internal_appserver|bool
+
+- name: Restart netbox Request Queue Worker
+  service:
+    name: 'netbox'
```

Comment:
It should probably be named `netbox-rq`.

Reply:
Yes, that makes more sense :) Fixed.